app/tests/spec/views/app.js

```diff
@@ -105,5 +118,316 @@ define([
         assert.isTrue(router.navigate.calledWith('signup'));
       });
     });
+
+    describe('setTitle', function () {
+      it('sets the document title', function () {
+        view.setTitle('Foo');
+        assert.equal(windowMock.document.title, 'Foo');
+      });
+    });
+
+    describe('showView', function () {
+      describe('with a view that does not render', function () {
+        var displayedView;
+        var isDestroyed = false;
+        var isLogged = false;
+
+        var DoesNotRenderView = Backbone.View.extend({
+          render: function () {
+            return p(false);
+          },
+
+          destroy: function () {
+            isDestroyed = true;
+          },
+
+          logScreen: function () {
+            isLogged = true;
+          }
+        });
+
+        before(function () {
```

Comment:
@philbooth - I was experimenting with different test styles in this module, closer to what I've seen you do. Is this closer to what you had in mind?